public/app/features/browse-dashboards/BrowseDashboardsPage.tsx

```diff
@@ -87,7 +87,9 @@ const BrowseDashboardsPage = memo(({ match }: Props) => {
 
   const hasSelection = useHasSelection();
 
-  const { canEditFolders, canEditDashboards, canCreateDashboards, canCreateFolders } = getFolderPermissions(folderDTO);
+  const { data: rootFolder } = useGetFolderQuery('general');
```

Comment:
@IevaVasiljeva maybe i "dumb" question. but will this always be 'general' . This is querying UUID?

```ts
    // get folder info (e.g. title, parents) but *not* children
    getFolder: builder.query<FolderDTO, string>({
      providesTags: (_result, _error, folderUID) => [{ type: 'getFolder', id: folderUID }],
      query: (folderUID) => ({ url: `/folders/${folderUID}`, params: { accesscontrol: true } }),
    }),
```

https://github.com/grafana/grafana/blob/main/public/app/features/browse-dashboards/api/browseDashboardsAPI.ts